File: StudMap/StudMap.Core/ResponseError.cs
namespace StudMap.Core
{
    public enum ResponseError
    {
        None = 0,

        #region General
        DatabaseError = 1,
        #endregion

        #region Register
        UserNameDuplicate = 101,
        UserNameInvalid = 102,
        PasswordInvalid = 103,
        #endregion

        #region Login
        LoginInvalid = 110,
        #endregion

        #region Maps
        MapIdDoesNotExist = 201,
        FloorIdDoesNotExist = 202,
        NodeIdDoesNotExist = 203,
        #endregion

        #region Navigation
        NoRouteFound = 301,
        StartNodeNotFound = 302,
        EndNodeNotFound = 303,
        #endregion

        #region Information
        PoiTypeIdDoesNotExist = 401,
        NFCTagDoesNotExist = 402,
        QRCodeDosNotExist = 403,
        PoiDoesNotExist = 404,
        #endregion // Information
    }
}

File: trunk/StudMap.Client.Core/src/de/whs/studmap/client/core/web/ResponseError.java
package de.whs.studmap.client.core.web;

public interface ResponseError {
			public static final int None = 0;

            public static final int DatabaseError = 1;
            
            
            public static final int UserNameDuplicate = 101;
            public static final int UserNameInvalid = 102;
            public static final int PasswordInvalid = 103;
            
            public static final int LoginInvalid = 110;

            public static final int MapIdDoesNotExist = 201;
            public static final int FloorIdDoesNotExist = 202;
}
